plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -605,6 +607,11 @@ fu_synaptics_rmi_device_setup (FuDevice *device, GError **error)
 	bl_ver = g_strdup_printf ("%u.0", priv->flash.bootloader_id[1]);
 	fu_device_set_version_bootloader (device, bl_ver);
 
+	/* determine if we're wired internally */
```

Comment:
that's a pretty good idea. I wonder if it makes sense to put this in `fu_udev_device_probe()` instead?

Reply:
Yes, good idea also.  Just pushed that.